packages/sso-menu/src/SSOMenu.tsx

```diff
@@ -0,0 +1,248 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import Button from '@leafygreen-ui/button';
+import Icon from '@leafygreen-ui/icon';
+import { Menu, MenuItem, MenuGroup } from '@leafygreen-ui/menu';
+import { uiColors } from '@leafygreen-ui/palette';
+import { css, cx } from '@leafygreen-ui/emotion';
+
+const menuButtonStyle = css`
+  height: 29px;
+  padding: 2px 14px;
```

Comment:
When 2px padding is removed from top/bottom the name no longer appears as centered. I can't tell if that's because it's truly off, or just appears as such. Will play around with this a bit.